src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -309,12 +309,22 @@ public AbstractGroup resultConverter(ButtonType button) {
                         searchGroupRegexProperty.getValue());
             } else if (typeAutoProperty.getValue()) {
                 if (autoGroupKeywordsOptionProperty.getValue()) {
+                    // Set default value for delimiters: ',' for base and '>' for hierarchical
+                    char delimiter = ',';
+                    char hierarDelimiter = '>';
```

Comment:
```suggestion
                    char hierarDelimiter = Keyword.DEFAULT_HIERARCHICAL_DELIMITER;
```

(org.jabref.model.entry.Keyword)